docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
+
+See the documentation for your particular flavor of Docker for instructions on configuring resource limits.
+
+### Example K8s container configuration
+
+The following Minikube example creates a K8s cluster named `k8ssandra` running K8s version 1.18.16 with 4 virtual processor cores and 8 gigs of RAM:
+
+```bash
+minikube start --cpus=4 --memory='8128m' --kubernetes-version=1.18.16 -p k8ssandra
+😄  [k8ssandra] minikube v1.17.1 on Darwin 11.2.1
+✨  Automatically selected the docker driver. Other choices: hyperkit, ssh
+👍  Starting control plane node k8ssandra in cluster k8ssandra
+🔥  Creating docker container (CPUs=4, Memory=8128MB) ...
+🐳  Preparing Kubernetes v1.18.16 on Docker 20.10.2 ...
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔎  Verifying Kubernetes components...
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "k8ssandra" cluster and "default" namespace by default
 ```
 
-Alternatively, you may download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+## Configure the K8ssandra Helm repository
+
+K8ssandra is delivered via a collection of Helm charts for easy installation, so once you've got a suitable K8s environment configured, you'll need to add the K8ssandra Helm chart repositories.
+
+To add the K8ssandra helm chart repos:
+
+1. Add the main K8ssandra Helm chart repo:
+
+    ```bash
+    helm repo add k8ssandra https://helm.k8ssandra.io/
+    ```
+
+2. If you want to access K8ssandra services from outside of the Kubernetes cluster, also add the Traefik Ingress repo (highly recommended):
+
+    ```bash
+    helm repo add traefik https://helm.traefik.io/traefik
+    ```
+
+3. Finally, update your helm repository listing:
+
+    ```bash
+    helm repo update
+    ```
+
+{{% alert title="Tip" color="success" %}}
+Alternatively, you can download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+{{% /alert %}}
 
 ## Install K8ssandra
 
-From a packaging perspective, K8ssandra is composed of a number of helm charts. It handles the installation of operators and custom resources as well as
-provisioning the cluster instances.
+The K8ssandra helm charts make installation a snap. You can override chart configurations during installation as necessary if you're an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
+
+{{% alert title="Important" color="warning" %}}
+K8ssandra comes out of the box with a set of default values tailored to getting up and running quickly.  Those defaults are intended to be a great starting point for smaller-scale local development but are **not** intended for production deployments.
```

Comment:
Add a link to `values.yaml` for the K8ssandra chart. Another option would be a link to the reference page for the chart.

```suggestion
K8ssandra comes out of the box with a set of [default values](https://github.com/k8ssandra/k8ssandra/blob/main/charts/k8ssandra/values.yaml) tailored to getting up and running quickly.  Those defaults are intended to be a great starting point for smaller-scale local development but are **not** intended for production deployments.
```

or


```suggestion
K8ssandra comes out of the box with a set of [default values]({{< ref "/docs/reference/k8ssandra/" >}}) tailored to getting up and running quickly.  Those defaults are intended to be a great starting point for smaller-scale local development but are **not** intended for production deployments.
```